src/chainstate/stacks/db/mod.rs

```diff
@@ -148,6 +148,7 @@ pub struct StacksHeaderInfo {
 pub struct StacksEpochReceipt {
     pub header: StacksHeaderInfo,
     pub tx_receipts: Vec<StacksTransactionReceipt>,
+    pub matured_rewards: Vec<MinerReward>,
```

Comment:
All of the `MinerReward`s in this vector will be matured rewards from the same block, so maybe it makes more sense to put the matured block's block hash and consensus hash in this struct directly, instead of duplicating it _n_ times?

Reply:
Sure